falcon/app.py

```diff
@@ -872,8 +872,8 @@ def _compose_status_response(self, req, resp, http_status):
             resp.set_headers(http_status.headers)
 
         # NOTE(kgriffs): If http_status.body is None, that's OK because
-        # it's acceptable to set resp.body to None (to indicate no body).
-        resp.body = http_status.body
+        # it's acceptable to set resp.text to None (to indicate no body).
+        resp.text = http_status.body
```

Comment:
shouldn't we also use `http_status.text`

Reply:
+1 for this. But please also retain `body` as a kwarg and also provide a `body` property for backward compatibility. The `HTTPStatus` docstring should mention that the kwarg and property are deprecated aliases and will be removed in a future release.